src/ActiveLogin.Authentication.BankId.AspNetCore/BankIdAuthenticationHandler.cs

```diff
@@ -139,7 +139,10 @@ protected override Task HandleChallengeAsync(AuthenticationProperties properties
 
             var loginOptions = new BankIdLoginOptions(
                 Options.BankIdCertificatePolicies,
-                null, 
+                properties.Items.TryGetValue(
```

Comment:
Can this be refactored into a method so it's clearer what it does? The onliner is a bit hard to get a grasp of at the moment.